enterprise/backend/test/metabase_enterprise/advanced_permissions/models/permissions_test.clj

```diff
@@ -0,0 +1,78 @@
+(ns metabase-enterprise.advanced-permissions.models.permissions-test
+  (:require [clojure.test :refer :all]
+            [metabase-enterprise.advanced-permissions.models.permissions :as ee-perms]
+            [metabase.models :refer [Permissions PermissionsGroup]]
+            [metabase.models.database :as database]
+            [metabase.models.permissions :as perms]
+            [metabase.public-settings.premium-features-test :as premium-features-test]
+            [metabase.test :as mt]
+            [metabase.util :as u]))
+
+(defn- test-download-perms [group-id]
```

Comment:
the name is somewhat confused to me, how about we call it `download-perms-by-group-id`?

Reply:
yeah, good call